icu4j/maven-build/maven-icu4j/pom.xml

```diff
@@ -0,0 +1,493 @@
+        <executions>
+          <execution>
+            <id>unpack-dependencies</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>unpack-dependencies</goal>
+            </goals>
+            <configuration>
+              <includeGroupIds>com.ibm.icu</includeGroupIds>
+              <!-- exclude test resource files which come through dependencies as `provided` scope -->
+              <excludeScope>provided</excludeScope>
+              <!--
+                Don't know why the Ant build.xml is configured to exclude `**/res_index.res` from the set of
+                included files since those files still appear in the packaged .jar file.
+              -->
+              <excludes>**/*.cnv,**/cnvalias.icu</excludes>
+              <outputDirectory>${project.build.directory}/classes</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- 
+        Process and transform the resource files before they get packaged up in the .jar file.
+      -->
+      <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>exec-maven-plugin</artifactId>
+          <version>3.1.0</version>
+          <executions>
+              <!-- 
+                Replicate the creation of fullLocaleNames.lst files in every subdirectory of the dir of unpacked data
+                that the Ant build.xml file encodes.
+              -->
+              <execution>
+                  <id>full-locale-names</id>
+                  <phase>process-resources</phase>
+                  <goals>
+                      <goal>exec</goal>
+                  </goals>
+                  <configuration>
+                      <executable>./file-listing-helper.sh</executable>
+                  </configuration>
+              </execution>
+              <!-- Copy the license file. -->
+              <execution>
+                  <id>copy-license-file</id>
+                  <phase>process-resources</phase>
+                  <goals>
+                      <goal>exec</goal>
+                  </goals>
+                  <configuration>
+                      <executable>cp</executable>
+                      <arguments>
+                        <argument>../../../icu4c/LICENSE</argument>
+                        <argument>${project.build.directory}/classes</argument>
+                      </arguments>
+                  </configuration>
+              </execution>
+          </executions>
+      </plugin>
+
+      <!--
+        Enable configuration of the Manifest file, etc. during the process of packaging the .jar file.
+
+        For future reference: https://stackoverflow.com/questions/38548271/difference-between-maven-plugins-assembly-plugins-jar-plugins-shaded-plugi
+      -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.3.0</version>
```

Comment:
I would try to move this to the parent (including the properties), in `build/pluginManagement/plugins`
And try to use it here and only override the few properties that are different between artifacts.

Reply:
Done.